twilight-gateway/Cargo.toml

```diff
@@ -51,11 +44,13 @@ tracing-subscriber = { default-features = false, features = ["fmt", "tracing-log
 
 [features]
 default = ["rustls-native-roots", "twilight-http", "zlib-stock"]
-native = ["dep:native-tls", "tokio-tungstenite/native-tls"]
-rustls-native-roots = ["dep:rustls-tls", "dep:rustls-native-certs", "tokio-tungstenite/rustls-tls-native-roots"]
-rustls-webpki-roots = ["dep:rustls-tls", "dep:webpki-roots", "tokio-tungstenite/rustls-tls-webpki-roots"]
+no-tls = ["tokio-websockets/sha1_smol"]
+native = ["tokio-websockets/native-tls", "tokio-websockets/openssl"]
+rustls-native-roots = ["tokio-websockets/rustls-native-roots"]
+rustls-webpki-roots = ["tokio-websockets/rustls-webpki-roots"]
 zlib-simd = ["dep:flate2", "flate2?/zlib-ng"]
 zlib-stock = ["dep:flate2", "flate2?/zlib"]
+websockets-simd = ["tokio-websockets/simd"]
```

Comment:
Is there ever a good reason to provide this as an optional feature? And if so, is there a good reason not to leave it on by default?

Reply:
Pulls in an extra dependency for accelerated UTF-8 validation. I left it off by default because simd-json and zlib-simd are off by default

Reply:
simd-json has much worse errors, questionable soundness, and platform incompatabilities compared to serde-json. zlib-simd requires CMake to build. I don't think simdutf8 has any such problems?

By having this as a separate feature results in a discoverability problem. We should help users help themselves to maximizing performance.

Reply:
Yeah simdutf8 is fairly compatible. On unsupported platforms or without any runtime SIMD support detected it just falls back to the std APIs, so it should be no problem to enable this.

Reply:
Then I'd like to always enable this feature.